index.html

```diff
@@ -169,10 +169,26 @@ <h5 class="media-text">LET’S CONNECT</h5>
                     <img src="./assets/Normal Button/Tertiary/Icons/Icon - Chevron - Right - Larger.svg">
                 </li>
                 <div class="line"></div>
-            </ul>
+
+            </div>
         </div>
 
     </section>
+    <section class="contact-section">
+        <h2 class="contact-heading">Contact me</h2>
+        <p class="contact-cta">If you have an application you are
+            interested in developing, a feature
+            that you need built or a project that
+            needs coding. I’d love to help with it </p>
+        <form action="#" class="contact-form">
+            <input type="text" id="name" placeholder="Yeremaias" maxlength="30" />
+            <input type="email" placeholder="natajayanj14@gmail.com" id="email" />
+            <textarea name="message" id="message" cols="25" rows="10" placeholder="Write your message here"></textarea>
```

Comment:
- [ ] The project requirements mention that the user cannot submit the form if any of the inputs are missing. Please make sure that all the inputs are required.
 - [ ] The message should have a limit of 500 characters. Please make sure that there is a limit of characters for this input as you did for the name input.